### PR TITLE
[WFCORE-7012][WFCORE-7013] Fix ServerEnvironmentTestCase and ignore when server is running in default stability level

### PR DESCRIPTION
**Pull request by yersan**
Jira issues:
https://issues.redhat.com/browse/WFCORE-7012
https://issues.redhat.com/browse/WFCORE-7013

### CODE REVIEW

**Comment by yersan** [comment]
Hello @michpetrov , could you review this? thanks

**Review by michpetrov — APPROVED** [review]
@yersan LGTM!

File: server/src/test/java/org/jboss/as/server/ServerEnvironmentTestCase.java [hunk 1]
```diff
@@ -104,6 +109,7 @@ public void testAliasFunctionality() throws IOException {
 
         // default stability = COMMUNITY
         ProductConfig productConfig = ProductConfig.fromFilesystemSlot(null, "", props);
+        Assume.assumeTrue(Stability.COMMUNITY.enables(productConfig.getDefaultStability()));
```
Comment (yersan) [inline server/src/test/java/org/jboss/as/server/ServerEnvironmentTestCase.java:112]: Ups, this should be `equals` not `enables`

**Comment by yersan** [comment]
Thanks @michpetrov

**Pull request merged**